File: javascript/bruteForceSort.js
let array = [];
let n = Math.floor(Math.random() * 100000);

for (let i = 0; i < n; i++) {
	array.push(Math.floor(Math.random() * 10000));
}

console.log("\nInitial array:\n" + array);

function bruteForceSort(A) {
	for (let i = 0; i < array.length; i++) {
		min = +Infinity; 
		for (let j = i; j < array.length; j++) {
			if (array[j] < min) {
				min = array[j];
				index = j;
			}
		}
		temp = array[i];
		array[i] = min; 
		array[index] = temp;
	}
}

var startTime = performance.now();
bruteForceSort(array);
var endTime = performance.now();

console.log("\nSorted array:\n" + array);
console.log("\nExecution time -> " + (endTime - startTime) + " ms");

File: python/bruteForceSort.py
import time
import random
import math

def bruteForceSort(A):
	for i in range(0, len(A) - 1):
		minimum = math.inf
		for j in range(i, len(A)):
			if A[j] < minimum:
				minimum = A[j]
				index = j
		temp = A[i]
		A[i] = minimum
		A[index] = temp


A = [random.randint(1, 10) for _ in range(10)]

print("Unsorted array:", A)

startTime = time.time()

bruteForceSort(A)

endTime = time.time()
executionTime = (endTime - startTime)

print("Sorted array:", A)
print(f"Execution time: {executionTime:.6f} seconds")